src/components/organisms/Footer.tsx

```diff
@@ -7,38 +7,36 @@ import { Typography } from 'components/atoms';
 import { silverSmokeColor, oceanBlueColor, skyBlueColor } from 'style';
 import ThankYouDialog from './ThankYouDialog';
 import AboutDialog from './AboutDialog';
+import { version } from '../../../package.json';
```

Comment:
no need to import relatively, you can just do this `import { version } from 'package.json'`

Reply:
this doesn't work for me. as far as I understand, absolute paths are available under paths listed as `include` in `tsconfig.json` ([here](https://create-react-app.dev/docs/importing-a-component/#absolute-imports)). Am I missing something?

Reply:
you can use absolute imports - `/package.json`

Reply:
as discussed, this requires changes to `tsconfig.json` or creating a symlink to `package.json` from within `/src`. For now, leaving the relative path as is